base/db/man/build_insert_query.Rd

```diff
@@ -11,7 +11,7 @@ build_insert_query(values, table, .con)
 
 \item{table}{Name of target SQL table, as character}
 
-\item{.con}{[\code{DBIConnection}]:A DBI connection object obtained from \code{DBI::dbConnect()}.}
+\item{.con}{Database connection object}
```

Comment:
I suspect this isn't a desirable change and I think there's a chance it's a Roxygen bug, but I'm not sure how to resolve it efficiently: Best I can tell it switched from inheriting `.con` from `glue::glue_sql` to inheriting `con` (no dot!) from `db.query`. @ashiklom I think you wrote these docs -- thoughts?

Reply:
I think it's fine. "Database connection object" is somewhat less precise than "DBI Connection", but not wrong, and to my knowledge, this function isn't that widely used anyway.